### PR TITLE
16756 Added special Court Order / POA handling in Consent for Continuation Out filing

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#16756

*Description of changes:*
- app version = 7.0.48
- bolded contact info labels (per design)
- renamed GenericErrorDialog -> AffiliationErrorDialog
- created GenericErrorDialog (same as Create UI)
- added new dialog, properties and logic to ConsentContinuationOut view
- make Court Order / POA section render for staff only
- added test suite for new dialog

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-629-dx09i8ap.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-629-dx09i8ap.web.app

File: src/components/common/ContactInfo.vue [hunk 1]
```diff
@@ -10,7 +10,7 @@
       >
         mdi-phone
       </v-icon>
-      <span class="px-2">Canada and U.S. Toll Free:</span>
+      <span class="px-2 font-weight-bold">Canada and U.S. Toll Free:</span>
```
Comment (severinbeauvais) [inline src/components/common/ContactInfo.vue:13]: Per Yui's latest design. (Also same as Create UI.)

File: src/components/dialogs/AffiliationErrorDialog.vue
Comment (severinbeauvais) [inline src/components/dialogs/AffiliationErrorDialog.vue:1]: This is the renamed component. I only changed the content-class and component name.

File: src/views/ConsentContinuationOut.vue [hunk 2]
```diff
@@ -403,9 +413,13 @@ export default class ConsentContinuationOut extends Mixins(CommonMixin, DateMixi
...
 
   // variables for displaying dialogs
+  courtOrderDialog = false
   resumeErrorDialog = false
-  saveErrorReason: SaveErrorReasons = null
+  saveErrorReason = null as SaveErrorReasons
   paymentErrorDialog = false
+  courtOrderDialogText = 'This business has a court order in its ledger. Please contact BC ' +
+    'Registries to submit a Consent to Continue Out.'
+  courtOrderDialogTitle = 'Please contact BC Registries'
```
Comment (severinbeauvais) [inline src/views/ConsentContinuationOut.vue:422]: These were a bit long for the declaration in the template so I created these properties instead.

File: src/components/dialogs/GenericErrorDialog.vue
[the remark this answers is not in the document]
Comment (severinbeauvais) [reply src/components/dialogs/GenericErrorDialog.vue:1]: I guess that, some day, we could move this component to the shared library, but that's not in scope for this ticket.

**Review by ketaki-deodhar — APPROVED** [review]
Looks good! Also, Covers Yui's comments 👍

**Pull request merged**